### PR TITLE
docs(*): add docs for inspect api

### PR DESCRIPTION
**Pull request by lobkovilya**
Docs for Inspect API

Fix: https://github.com/kumahq/kuma-website/issues/683

### CODE REVIEW

File: docs/docs/dev/documentation/inspect-api.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+Get config dump for zone-egress:
+```shell
+kumactl inspect zoneegress ze-1 --config-dump
+```
+
+::: warning
+In order to get config dump in Multizone deployment `kumactl` should be pointed to Zone CP, today Global CP doesn't have access 
+to envoy config dumps. This is a limitation, and we're going to resolve it in the next releases.
```
Comment (lahabana) [inline docs/docs/dev/documentation/inspect-api.md:95]: Can you link to the github issue please :)

Comment (lobkovilya) [reply docs/docs/dev/documentation/inspect-api.md]: Done

File: docs/docs/dev/documentation/http-api.md [hunk 2]
```diff
@@ -3390,3 +3393,69 @@ curl localhost:5681/meshes/default/health-checks/gateway-to-backend/policies
...
+ "configs": [
+  {
+   "@type": "type.googleapis.com/envoy.admin.v3.BootstrapConfigDump",
+   "bootstrap": {
+    "node": {
+     "id": "default.backend-1",
+     "cluster": "backend",
+     "metadata": {
+       "dataplane.admin.port": "6606",
+...
```
[the remark this answers is not in the document]
Comment (lobkovilya) [reply docs/docs/dev/documentation/http-api.md]: Done

File: docs/docs/dev/documentation/inspect-api.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- Outbound – policy is applied to envoy outbound listener
+- Service – policy is applied to envoy outbound cluster (upstream cluster)
+- Dataplane – the area where policy applied is not specified, could affect inbound/outbound listeners and clusters
+
+The command in the example above shows what policies were matched for every type of attachment. 
+
+## Affected data plane proxies
+
+Sometimes it's useful to see if it's safe to delete or modify some policy. Before doing any critical changes
+it is worth checking what data plane proxies will be affected. This could be done using Inspect API as well:
```
[the remark this answers is not in the document]
Comment (lobkovilya) [reply docs/docs/dev/documentation/inspect-api.md:54]: This use case is kind of a sub-use case of what's written here: `it is worth checking what data plane proxies will be affected` means you can check if `none data plane proxies will be affected`. WDYT?

**Pull request merged**